pyproject.toml

```diff
@@ -30,7 +30,6 @@ ignore = [
     "B007", "B008", "B017",
     "B018", # Useless expression
     "B019", "B020",
-    "B022", # Allow empty context manager
```

Comment:
remove the ignore from .flake8 file as well.

Reply:
There was not an entry in flake8 for some reason.

Reply:
Interesting...

Reply:
Potentially the bugbear version we use is an older one